vignettes/UsingSelfControlledCohort.Rmd

```diff
@@ -0,0 +1,222 @@
+---
+title: "Using SelfControlledCohort"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Using SelfControlledCohort}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+
+The Self Controlled Cohort method measures the association between an exposure and an outcome by comparing the number of outcomes during an unexposed time at risk to the number of outcomes durning an exposed time at risk. The method is called "Self-Controlled" because each individual in the study contributes person-time to both the exposed and unexposed cohorts. There are seven time points to keep in mind when defining this analysis shown in the diagram below.
+
+```{r, echo=FALSE}
+  # Note over Tn100,T0: washout period
+  # Note over T0,T100: Followup period
+  # 
+library(DiagrammeR)
```

Comment:
The DiagrammeR package needs to be added to the `Suggests` section of the DESCRIPTION file (or else the CI fails)